analysis/druid/src/core/ActiveDruidForm.ts

```diff
@@ -0,0 +1,74 @@
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+
+export default class ActiveDruidForm extends Analyzer {
+
+    private currentForm: DruidForm = "No Form";
+
+    constructor(options: Options) {
+        super(options);
+        this.active = true
```

Comment:
The module is active by default if it's loaded, this can be removed.

Reply:
Thanks for the info, removed.